src/data/storage.js

```diff
@@ -15,21 +17,25 @@ export const LIFESPAN = Object.freeze({
   LONG,
 })
 
-export const save = (key, value, LIFESPAN) => LIFESPAN.setItem(key, value)
+export const save = (key, value, lifespan) => {
+  Object.keys(LIFESPAN).filter(type => LIFESPAN[type] !== lifespan).map(type => {
```

Comment:
For readability, perhaps extract `Object.keys(LIFESPAN)` to a method named `getLifespanTypes()`

Reply:
np:  https://github.com/silinternational/wecarry-ui/commit/46ee31024e77c1b4870976b9dfc635ac2d8dc7e9